src/DssSpell.t.sol

```diff
@@ -507,7 +507,7 @@ contract DssSpellTest is DssSpellTestBase {
         uint256 finPlanned;
     }
 
-    function testYankDAI() private { // make private to disable
+    function testYankDAI() public { // make private to disable
```

Comment:
As [specified in the sheet](https://docs.google.com/spreadsheets/d/1w_z5WpqxzwreCcaveB2Ye1PP5B8QAHDglzyxKHG3CHw/edit#gid=1031558994&range=B56), one of the streams is on `MCD_VEST_DAI_LEGACY` contract. So you would need to duplicate the test and modify it (see line 525) in order to check both yanks.